### PR TITLE
New: [T1848] Merge "bank card" & "bank account" tab 

### PR DESCRIPTION
**Pull request by aspangaro**
New: Removed state_id in bank account
New: Add an option to hide RIB information

### CODE REVIEW

File: htdocs/compta/bank/class/account.class.php [hunk 1]
```diff
@@ -54,14 +55,6 @@ class Account extends CommonObject
     var $clos;
     var $rappro=1;    // If bank need to be conciliated
     var $url;
-    //! BBAN field for French Code banque
-    var $code_banque;
```
Comment (marcosgdf) [inline htdocs/compta/bank/class/account.class.php:58]: ¿?

Comment (aspangaro) [inline htdocs/compta/bank/class/account.class.php:58]: For french people, four fields exists for RIB system. (code_banque | code_guichet | number | cle_rib)

This system is deprecated from august 2014. It's replaced by IBAN/BIC system in European Area.

Comment (marcosgdf) [inline htdocs/compta/bank/class/account.class.php:58]: What about old data? In spain we use CCC: Bank code, Branch code, control code and Account number. We officially use IBAN but people still use CCC nowadays

Comment (aspangaro) [inline htdocs/compta/bank/class/account.class.php:58]: Your CCC system is like RIB system, your control code is before the account number, for us, in france, it's after the account number.

Normally, this system is deprecated more than 1 year when this version was released in stable version.

**Pull request closed without merging**